suite-native/module-settings/src/navigation/SettingsStackNavigator.tsx

```diff
@@ -13,6 +13,7 @@ import { SettingsLocalisationScreen } from '../screens/SettingsLocalisationScree
 import { SettingsCustomizationScreen } from '../screens/SettingsCustomizationScreen';
 import { SettingsAnalyticsScreen } from '../screens/SettingsAnalyticsScreen';
 import { SettingsAboutUsScreen } from '../screens/SettingsAboutUsScreen';
+import { SettingsFAQ } from '../components/SettingsFAQ';
```

Comment:
Is there a reason why is SettingsFAQ in the `components` folder and not in the `screens`?

Move it to the `screens` folder and add `Screen` suffix.

Reply:
https://github.com/trezor/trezor-suite/pull/7792/commits/c595ab010953f51589666bab9f8a90f3dfa4cc36